client/verta/verta/deployment/autoscaling/metrics.py

```diff
@@ -47,13 +47,20 @@ class CpuUtilizationTarget(_AutoscalingMetric):
     """
     CPU utilization target to trigger autoscaling.
 
+    The JSON equivalent for this is:
+
+    .. code-block:: json
+
+        {
+            "metrics": [{"metric": "cpu_utilization", "parameters": [{"name": "target", "value": 0.5}]}]
+        }
```

Comment:
1. I think this should just be the single metric itself, so this can also be copy&paste as a CLI argument
1. Could you also mention (as part of the line above, "The JSON equivalent for this is:") that this is meant to be an element of autoscaling metrics JSON?
    - same thing for update rules

```suggestion
        {"metric": "cpu_utilization", "parameters": [{"name": "target", "value": 0.5}]}
```

Reply:
Done.